packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx

```diff
@@ -167,13 +167,14 @@ export const DataSourceHttpSettings: React.FC<HttpSettingsProps> = (props) => {
               <div className="gf-form">
                 <FormField
                   label="Timeout"
+                  type="number"
```

Comment:
nit this might not have any affect in Safari/FireFox (see https://github.com/grafana/grafana/pull/29558) but no worries as we're using parseInt in the onChange.